lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -11,9 +11,12 @@ class ProcessRedfishAlert < Filter
     config_param :coloregion, :string
     config_param :username, :string
     config_param :passwordFile, :string
+    config_param :hardware, :string, :default => "SDFLEX"
```

Comment:
can we add a check on the hardware type as it comes in, and fail if it is not SDflex or dell